src/Microsoft.Azure.ServiceBus/MessageHandlerOptions.cs

```diff
@@ -62,7 +62,8 @@ public int MaxConcurrentCalls
         public bool AutoComplete { get; set; }
 
         /// <summary>Gets or sets the maximum duration within which the lock will be renewed automatically. This
-        /// value should be greater than the longest message lock duration; for example, the LockDuration Property. </summary>
+        /// value should be greater than the longest message lock duration; for example, the LockDuration Property. 
+        /// Also client must have manage rather than just listen permissions (no error will be thrown)</summary>
```

Comment:
Sounds like a \<remark\>. With a remark you would not need the "also".

Additional info could be "renewal is not a guaranteed operation and can fail".

Reply:
@paulgmiller ping